std_msgs/Header header
common/Uint8TimeStamped cdcu_eps_workmode
common/FloatTimeStamped cdcu_eps_strtrq
common/FloatTimeStamped cdcu_eps_strwhlangle
common/FloatTimeStamped cdcu_eps_whlspd
common/Uint8TimeStamped cdcu_eps_errlevel